app/digital/ts/models/ioobjects/gates/ORGate.ts

```diff
@@ -24,6 +28,41 @@ export class ORGate extends Gate {
         return "or.svg";
     }
 
+    public getMinPos(): Vector {
+        const min = super.getMinPos();
+
+
+        var BOX_WIDTH = DEFAULT_BORDER_WIDTH;
+        if (this.numInputs() >3 && this.numInputs() < 7)
```

Comment:
Can we get a comment or two explaining this behavior? It's not super obvious why this is necessary.

Also let's put a space between the `>` and the `3`